### DISCUSSION TITLE
Bug:  【偶发现象】ERROR  [h3] [unhandled] Cannot read properties  of null (reading 'stack')

### DISCUSSION THREAD

**Discussion by M69W** [vbenjs/vue-vben-admin#4460] [General]
### Version

Vben Admin V5

### Describe the bug?

输入 pnpm dev
偶发出现 以下描述，
需要稍等一会，本地 api (backend-mock)的服务才正常，影响程度极低
只是偶发现象
```
[nitro] rollup:before: 0.185ms
[nitro] types:extend: 0.027ms

[14:36:00] ✔ Nitro Mock Server started.

 ERROR  [h3] [unhandled] Cannot read properties
 of null (reading 'stack')

  at errorHandler (F:/xxx/apps/backend-mock/error.ts:4:47)
  at Object.handler (F:/xxx/node_modules/.pnpm/nitropack@2.9.7_encoding@0.1.13/node_modules/nitropack/dist/chunks/server.mjs:365:16)
  at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  at async Object.handler (F:\xxx\node_modules\.pnpm\h3@1.12.0\node_modules\h3\dist\index.cjs:1982:19)
  at async Server.toNodeHandle (F:\xxx\node_modules\.pnpm\h3@1.12.0\node_modules\h3\dist\index.cjs:2273:7)

[nitro 14:36:07] ✔ Nitro Server built in 10151 ms
[nitro] compiled: 1.059ms
[nitro] dev:reload: 280.618ms
```
![偶发现象](https://github.com/user-attachments/assets/e312f803-ad18-4cb1-9482-d8f463d6d2e7)

### Reproduction

偶发现象，并不一定能重现

### System Info

```shell
System:
    OS: Windows 10
    CPU: (8) x64 Intel(R) Core(TM) i5
  Binaries:
    Node: 20.16.0
    Yarn: 1.22.22
    npm: 10.8.1
    pnpm: 9.6.0
```

### Validations

- [X] Read the [docs](https://anncwb.github.io/vue-vben-admin-doc/)
- [X] Ensure the code is up to date. (Some issues have been fixed in the latest version)
- [X] I have searched the [existing issues](https://github.com/vbenjs/vue-vben-admin/issues) and checked that my issue does not duplicate any existing issues.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vbenjs/vue-vben-admin/discussions) or join our [Discord Chat Server](https://discord.gg/8GuAdwDhj6).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by mynetfan** [comment]
这是mock server报的错。在mock server启动过程中遇到需要处理的请求时就大概率会出现这个错误。等mock server完全启动之后再发起请求就不会有这种错误了。
重现方法：运行 `pnpm dev:play`启动开发服务，浏览器打开[登陆过期测试页面](http://localhost:5555/demos/features/login-expired)。此时关闭开发服务重新运行`pnpm dev:play`，在启动过程中快速去之前浏览器打开的页面里点击”点击触发“按钮以便向mock server发起一个请求。此时就会因为mock server已经开始启动但尚未准备就绪抛出你所截图的错误。

**Comment by mynetfan** [comment]
在控制台出现`Nitro Server built in xxxx ms`这条消息之后就这个错误就不会出现了

**Comment by M69W** [comment]
> 这是mock server报的错。在mock server启动过程中遇到需要处理的请求时就大概率会出现这个错误。等mock server完全启动之后再发起请求就不会有这种错误了。 重现方法：运行 `pnpm dev:play`启动开发服务，浏览器打开[登陆过期测试页面](http://localhost:5555/demos/features/login-expired)。此时关闭开发服务重新运行`pnpm dev:play`，在启动过程中快速去之前浏览器打开的页面里点击”点击触发“按钮以便向mock server发起一个请求。此时就会因为mock server已经开始启动但尚未准备就绪抛出你所截图的错误。

重现过程有印象，确定如此。
之前在微信群反馈过，
![thumb_172673775760933 png](https://github.com/user-attachments/assets/315c6a63-2612-4854-b2d2-16fcae6182a8)
就当做一个记录吧
